docs/reference/query-dsl/script-query.asciidoc

```diff
@@ -4,12 +4,16 @@
 <titleabbrev>Script</titleabbrev>
 ++++
 
-A query allowing to define
-<<modules-scripting,scripts>> as queries. They are typically used in a filter
-context, for example:
+
+Accepts another query as a <<modules-scripting-using,script>>. `script` queries
+are typically used in a <<query-filter-context,filter context>>.
+
```

Comment:
> Accepts another query as a <<modules-scripting-using,script>>

It doesn't really accept another query.  I would rather say as it was before or something like: 

"`script_query` is used to filter documents based on the provided script. They are typically used in a filter context."

Reply:
Thanks @mayya-sharipova! Fixed with 08bc253.